controllers/users.js

```
@@ -1,4 +1,29 @@
-const { User } = require('../models');
+const bcrypt = require('bcrypt');
+const { User } = require('../models'); 
+const { validationResult } = require('express-validator');
+
+const authUser = async (req, res) => {
+  const errors = validationResult(req);
+  const {Email} = req.body;
+  
+  if(!errors.isEmpty()){
+    res.send('ok: false');
+  }else{
+    const userFinded = await User.findOne({ where: {email: Email}});
```

Es probable que se lleve este código a otro archivo en caso de no ser asi lo comun es validar en la linea siguiente a la petición y realizar un Fail Fast, sería solo un if con un return de lo que queres

PD: Intentar siempre responder un JSON (Ver REST)

---

Corregido